components/RuiGrid/tests/RuiGrid.test.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint @typescript-eslint/explicit-function-return-type: 0 */
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import '../src';
+import { TestUtils } from '../../../TestUtils';
+
+describe("RuiGrid", () => {
```

Comment:
Do you mind updating all tests to make sure we are using single quotes for `describe` and `it` please

Reply:
I know you copied my test which was good, you just copied my mistake soz

Reply:
sure thing! will also have to investigate my formatter!